doc/source/ray-core/compiled-graph/advanced_user_guide.rst

```diff
@@ -0,0 +1,14 @@
+Advanced User Guide
+===================
+
+Contents
+- Multi Output Cases (TP)
+- Overlap GPU to GPU comm (PP)
+- static GPU to GPU shape (PP)
+- Failures Semantics
+- Thread safety
+- Debugging tips
+    - remote debugger
```

Comment:
```suggestion
    - Remote debugger
```

Reply:
Btw, I will remove these parts before merging this PR. And create another one so that you guys can follow up.